module-apps/apps-common/ApplicationCommon.hpp

```diff
@@ -33,20 +33,33 @@
 #include <locks/handlers/SimLockSubject.hpp>
 #include <locks/handlers/LockPolicyHandler.hpp>
 #include "WindowsFactory.hpp"
-#include "WindowsStack.hpp"
+#include "WindowsPopupQueue.hpp"
+#include "PopupBlueprintFactory.hpp"
+#include "AppSwitchReason.hpp"
+#include "AppStartupReason.hpp"
 
 namespace app
 {
     class WindowsStack;
-
+    typedef std::function<bool(sys::Message *, const std::string &name)> UiNotificationFilter;
```

Comment:
Hmm using ? we mostly use "using" or there is some difference ?

Reply:
will change to using (using count: 100, typedef count 1 - I lost)